### DISCUSSION TITLE
Where is "Use this Template" button?

### DISCUSSION THREAD

**Discussion by Anneqxy** [microsoft/SATechnicalOnboarding#91] [Questions]
Hello, I am new here. When I was doing Section 1: Create your own JavaScript Portfolio Site with GitHub Codespaces and Copilot
, I didn't know where is "Use this Template" button. Does someone know it?
<img width="813" alt="截屏2024-01-17 上午11 07 07" src="https://github.com/microsoft/SATechnicalOnboarding/assets/113318686/1a73c715-c49e-40f2-987f-4b2867fcea37">

**Comment by theoyinbooke** [comment]
Hello @Anneqxy ,

I am also new to the program and I have similar question while going through this guide.

Here is what I did.
1. Make sure you are logged in to GitHub.
2. You will see the "Use this Template" button at the top right of the page where you are reading the instructions as can be seen below

![image](https://github.com/microsoft/SATechnicalOnboarding/assets/13628137/4696fa60-191c-41a4-9816-5749288bcc96)

**Reply by AnthonyByansi** [reply]
Thanks @theoyinbooke for your assistance! Highly appreciate it!

**Reply by theoyinbooke** [reply]
Happy to help.
